awx/api/generics.py

```diff
@@ -512,6 +512,9 @@ class SubListAPIView(ParentMixin, ListAPIView):
     # And optionally (user must have given access permission on parent object
     # to view sublist):
     #   parent_access = 'read'
+    # filter_read_permission sets whether or not to override the default intersection behavior
+    # implemented here
```

Comment:
This reads a little funny. I'd say drop `implemented here` or merge it with the line above.

Reply:
fixing